src/components/Artifact.js

```diff
@@ -84,17 +96,23 @@ export default function Artifact(props) {
           <Row className='row-mini-header'>
             <Col sm><span className='mini-header'>Artifact Entries</span></Col>
           </Row>
-          {<ul>
-            {artifact_entries.map(e => <li><Row key={e}><Col sm>{e}</Col></Row></li>)}
-          </ul>}
+            {artifact_entries.length <1 ? <p>No entries</p> : <ul>
+              {artifact_entries.map(e => <li><Row key={e}><Col sm>{e}</Col></Row></li>)}
+            </ul>}
           <Row>
-            <Col sm><span className='mini-header new'>Plots</span></Col>
+            <Col sm><span className='mini-header'>Plots</span></Col>
           </Row>
-          {<span className="new"><ul><li>Not part of a plot</li><li>Link to plot 1</li><li>Link to plot 2</li></ul></span>}
+            {artifactDetails.plots.length<1 ? <p>No linked plots</p> : <ul> {artifactDetails.plots.map(p => <li>
+                  <Link onClick={() => props.changeTab('Plots')} to={`/plots/${p.id}`}>{p.name}</Link></li>)}
+                  </ul>
+            }
           <Row>
-              <Col sm><span className='mini-header new'>Events [CREATE EVENT BUTTON]</span></Col>
+              <Col sm><span className='mini-header'>Events <span className='new'>[CREATE EVENT BUTTON]</span></span></Col>
             </Row>
-            {<span className="new"><ul><li>Not part of an event</li><li>Link to event</li></ul></span>}
+            {artifactDetails.events.length<1 ? <p>No linked events</p> : <ul> {artifactDetails.events.map(e => <li>
+                  <Link onClick={() => props.changeTab('Events')} to={`/events/${e.id}`}>{e.name}</Link></li>)}
+                  </ul>
+            }
           <Row>
             <Col sm><span className='mini-header new'>GM Notes</span></Col>
```

Comment:
Just add GM Notes ("gm_notes" can be found from ​/science​/artifact​/{id} request response) similarly as other data.